pkg/nfd-master/nfd-master.go

```diff
@@ -905,11 +905,11 @@ func (m *nfdMaster) setTaints(taints []corev1.Taint, nodeName string) error {
 	}
 
 	if taintsUpdated {
-		err = controller.PatchNodeTaints(context.TODO(), m.k8sClient, nodeName, node, newNode)
+		err := controller.PatchNodeTaints(context.TODO(), m.k8sClient, node.Name, node, newNode)
```

Comment:
nit
```suggestion
		err = controller.PatchNodeTaints(context.TODO(), m.k8sClient, node.Name, node, newNode)
```

Reply:
It didn't really matter. I prefer to use the narrowest scope for variables, though. Changed this (and another similar place) to use the `if err := ...` construct

@TessaIO PTAL

Reply:
Yes, I got it, I thought it would be better to be consistent.